.github/workflows/ci.yaml

```diff
@@ -20,10 +20,35 @@ env:
 
   GO_VERSION: "1.19"
   PYTHON_VERSION: "3.8.10"
-  NODE_VERSION: "16"
-  NPM_VERSION: "8.19.2"
+  NVM_VERSION: "0.39.3" # Does not need to be updated as often as the Node.js version or the NPM version
 
 jobs:
+
+  # ---- Configuration ----
+  config:
+    name: "Configuration"
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install nvm and Node.js
+        run: |
+          curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v${{ env.NVM_VERSION }}/install.sh | bash
+          export NVM_DIR="$HOME/.nvm"
+          [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+          nvm install "$(cat .nvmrc)"
+          
+      - name: Archive nvm and Node.js
```

Comment:
What is thepurpose of archiving and uploading? Pleas leave a comment in the action why this is done.

Reply:
It seemed more efficient to do the installation process of nvm only once and upload it as an archive. This can later be imported and extracted quickly in the hooks/scanners testing containers. It also ensures that we pull from  an external source to install nvm only once, instead of _**x**_ number of hooks + _**y**_ number of scanners

Reply:
You should write exactly this as a comment into the code.